params.go

```diff
@@ -51,7 +51,7 @@ var (
 	foragingDeerMaxPerHunt = flag.Uint(
 		"foragingMaxDeerPerHunt",
 		4,
-		"Max possible number of deer on a single hunt (regardless of number of participants).",
+		"Max possible number of deer on a single hunt (regardless of number of participants). ** should be strictly less than max deer population.",
```

Comment:
you should add a check in the `parseConfig` function which now allows you to return parse errors.

Reply:
if you have this, you can remove your above check (i'd still leave it in)

Reply:
Good idea - will do soon